src/main/java/frc/robot/RobotContainer.java

```diff
@@ -20,7 +18,6 @@
   // The robot's subsystems and commands are defined here...
   private final ExampleSubsystem m_exampleSubsystem = new ExampleSubsystem();
```

Comment:
Let's get rid of this and the `ExampleSubsystem` java file. Don't forget to remove the import

Reply:
did you want me to git rid of ExampleCommand file also